.travis.yml

```diff
@@ -2,20 +2,24 @@ language: java
 addons:
   apt:
     sources:
-    - r-packages-precise
+      - r-packages-precise
     packages:
-    - r-base
+      - r-base
 jdk:
-- oraclejdk8
+  - oraclejdk8
 env:
   matrix:
-  - TERM=dumb
+    - TERM=dumb
+    - TERM=dumb DATAFLOW_RUNNER=SparkPipelineRunner
```

Comment:
Reading the documentation a bit more, and looking at various examples it looks like the allow_failures subsection is specified outside the matrix. The env/matrix/allow_failures combination is not allowed by `travis lint`.

However, it turns out that I got the allow_failures part wrong before as I had 

```
matrix:
  allow_failures:
    - TERM=dumb DATAFLOW_RUNNER=SparkPipelineRunner
```

rather than

```
matrix:
  allow_failures:
    - env: TERM=dumb DATAFLOW_RUNNER=SparkPipelineRunner
```

(Note the addition of `env:`)

This is now working - you can see the "Allowed Failures" section on https://travis-ci.org/broadinstitute/hellbender/builds/68285275.